src/components/Pagination/Pagination.tsx

```diff
@@ -80,17 +90,15 @@ export default class Pagination extends React.Component<PaginationProps> {
         <Link
           attributes={{ ...pageAttributes }}
           className={bem("link", modifiers, "pagination", [pageClass])}
-          href="#"
+          href={changeUrls ? getPageHref(item) : "#"}
```

Comment:
Could we please try setting `pointer-events: none` on the highlighted page?  That will help to reinforce that that's the page you are on.  Right now, with it just being a different color, it can be mistaken at a glance for `a:visited {color: black;}` . Doing this will also save a trip to the router [if the user happened to click on the page he/she was already on].